### DISCUSSION TITLE
Directory is not outputting using @babel/cli in build while not using ignore files flag

### DISCUSSION THREAD

**Discussion by aroyan** [babel/babel#14559] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43630681/168734909-87e71332-97bc-474c-96f4-870250bbc663.png)
Hello everyone, I'm having this folder structure in the picture. I want to build `src` folder to `build` folder with command `npx babel src --out-dir build` but the views folder and file inside it from src is not getting built. 

And this is my `.babelrc` file
```json
{
  "presets": ["@babel/preset-env"],
  "plugins": [
    [
      "module-resolver",
      {
        "root": ["./src"]
      }
    ]
  ]
}
```

This is the video while I'm running the command

https://user-images.githubusercontent.com/43630681/168736925-d0325eee-2150-4304-9223-5ab5444653a9.mp4

How to solved this? Thank you

**Comment by liuxingbaoyu** [comment, marked as answer]
npx babel src --out-dir dist --copy-files

**Reply by aroyan** [reply]
Thank you, fixed my problem
